modules/exploits/freebsd/local/rtld_execl_priv_esc.rb

```diff
@@ -145,17 +148,17 @@ def exploit
 
     payload_path = "#{base_dir}/.#{rand_text_alphanumeric 5..10}"
 
-    executable_data = <<-EOF
-#include <stdio.h>
-#include <stdlib.h>
-#include <unistd.h>
+    executable_data = <<~EOF
```

Comment:
presumably these both do the same thing?

Reply:
It's the squiggly heredoc syntax, `<<~` which removes indentation for you.

https://www.rubyguides.com/2018/11/ruby-heredoc/
https://docs.ruby-lang.org/en/2.6.0/syntax/literals_rdoc.html#label-Here+Documents

So in this scenario, yip it should be good :+1: